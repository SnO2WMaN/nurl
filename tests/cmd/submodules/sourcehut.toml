args = ["https://git.sr.ht/~cnx/blackshades", "2.5.0", "--submodules"]
