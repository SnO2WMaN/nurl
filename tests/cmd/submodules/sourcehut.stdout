fetchFromSourcehut {
  owner = "~cnx";
  repo = "blackshades";
  rev = "2.5.0";
  hash = "sha256-cH9o1+43TvZ96SmPRDS5cLAzVSf16qohhKDgjHj8cQc=";
  fetchSubmodules = true;
}